OKTA-620014 Updates to Web App with Redirect Quickstart (#4295)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/sign-into-web-app-redirect/main/php/testapp.md

1. Start your app with the built-in PHP server:

   ```bash
   php -S 127.0.0.1:8080 -t public
   ```

1. Open a browser and go to `http://localhost:8080`.
1. Click **Sign In**. The browser redirects you to Okta to sign in using the Sign-In Widget.
1. After you’ve signed in, check your user's name appears.

> **Note**: If you’re signed in as an administrator in the same browser already, it displays your name. You can open an incognito window and create a test user in the Admin Console to use if you want.

### Troubleshooting

When troubleshooting errors in `authorization_code_callback_handler()`, use `var_dump()` to echo the response to a token request to the page. For example:

```php
if(isset($response['error'])) {
    var_dump($response);
    throw new Exception("token endpoint returned an error: ".$response['error']);
}

if(!isset($response['access_token'])) {
    var_dump($response);

>     throw new Exception("token endpoint did not return an error or an access token");

Reviewer: ```suggestion
    throw new Exception("token endpoint didn't return an error or an access token");
```

}
```

> Use the [/token](/docs/reference/api/oidc/#token) reference docs to understand the issue. If the response is `NULL`, echo the response from the curl call to the page:

Reviewer: ```suggestion
Use the [/token](https://developer.okta.com/docs/api/openapi/okta-oauth/oauth/tag/CustomAS/#tag/CustomAS/operation/tokenCustomAS) reference docs to understand the issue. If the response is `NULL`, echo the response from the curl call to the page:
```

Author: @susanharper-okta  WIll leave this URL as is until the new page has all the info from the old one.
